contracts/base/NFTDrop.sol

```diff
@@ -0,0 +1,307 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+//  ==========  External imports    ==========
+
+import "@openzeppelin/contracts/utils/Multicall.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "@openzeppelin/contracts/interfaces/IERC2981.sol";
```

Comment:
this one is not copied, can we copy it and add it to our `preset` folder like we do for `ERC27771Context.sol` ? 

that would remove all dependencies on external openzeppelin contracts